app/jobs/upload_uphold_access_parameters_job.rb

```diff
@@ -0,0 +1,11 @@
+class UploadUpholdAccessParametersJob < ApplicationJob
```

Comment:
Rails specific question: Is there a UI available for viewing job status? ex: job log, including params passed and status (did an exception occur, etc)